browser/src/control/Control.JSDialog.js

```diff
@@ -850,6 +855,17 @@ L.Control.JSDialog = L.Control.extend({
 	},
 
 	updatePosition: function (target, newX, newY) {
+		var width = target.getBoundingClientRect().width;
```

Comment:
This function is a setter (only applies the calculated position) - is used not only in autofilter - see `onPan` just above.

Can we separate these two cases? And use new calculation only for auto popups, but onPan and all other old cases will use only setter?
```
updatePosition() {
    here your caclulation code...
    setNewPosition(target, x, y);
}

setNewPosition(target, x, y) {
    here the old code....
}
```